functests/deploy-manager.go

```diff
@@ -0,0 +1,36 @@
+package functests
```

Comment:
Consider renaming this file to client.go?

Reply:
As I am working on more k8s objects, this file will have more common code in it so I will be renaming it to common.go

Reply:
Renamed it to common.go as it will have more common code.